### ISSUE TITLE
Fix 'from' headers from labs, and edit custom email template

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/lookit-api#1356]
# TL; DR

The use of lab email addresses as the "From" address in custom messages is causing deliverability problems. The domain used in the 'from' address in all emails we send needs to match the domain that we're actually sending from (currently 'mit.edu', though we could change this in the future).

# Narrative

Custom emails sent by researchers using BC or Berkeley domains for the lab email addresses are already being blocked (not delivered) by email clients (due to strict DMARC policies for those domains). And the new email deliverability requirements for Gmail say that we "must not impersonate From: headers", which means that emails sent from researchers using a Gmail address as their lab account will also be blocked.

We can probably fix this by sending out the custom emails from our mit.edu domain, hopefully using the lab's email as the reply-to address. We still need to test this solution to confirm, but SendGrid actually recommends this as a solution (see [here](https://support.sendgrid.com/hc/en-us/articles/360041356934-Troubleshooting-Email-Delivery-Failures-due-to-DMARC)), so I believe it will work.

From the family's perspective, this change will mean that, instead of seeing study-related emails sent from specific labs, they will see start seeing those emails as sent from lookit-bot@mit.edu, i.e. the same address that sends the automated study announcement emails. If this change will be confusing, we can add some automatic text to the email templates (text and HTML) for emails sent from researchers. Something like "This message was sent on behalf of [LAB/RESEARCHER] regarding [STUDY NAME]. If you'd like to reply, feel free to contact [LAB/RESEARCHER] or reply directly to this email." @ianchandlercampbell I'll let you specify exactly what this text should say and where it should go in the email content. 

Here are our existing templates for custom emails:
- html: https://github.com/lookit/lookit-api/blob/develop/studies/templates/emails/custom_email.html
- text: https://github.com/lookit/lookit-api/blob/develop/studies/templates/emails/custom_email.txt

Once the unsubscribe PR is merged, those templates will include two links at the bottom ("Update your email preferences" and "Unsubscribe from all emails", [see changes here](https://github.com/lookit/lookit-api/pull/1349/files#diff-66d11538a1941e93e85ae74a8d7b7293c08701cce2a571868b938efe0f8a27e8)).

I don't think anything should change from the researcher's perspective - @ianchandlercampbell please let me know if I'm wrong about that.

Relevant links:

https://sendgrid.com/en-us/blog/gmail-yahoo-sender-requirements#5-dont-impersonate-gmail-from-headers

https://sendgrid.com/en-us/blog/gmail-yahoo-sender-requirements#8-for-direct-mail-the-domain-in-the-senders-from-header-must-be-aligned-with-either-the-spf-domain-or-the-dkim-domain

**Comment by ianchandlercampbell** [comment]
Good to hear that SendGrid officially recommends the solution you were thinking about! I think this does not change anything from the researcher's perspective and perhaps even clarifies things from the family's perspective (that messaging is initially handled through CHS).

**Comment by ianchandlercampbell** [comment]
@becky-gilbert Here's a quick email draft! I've placed comments on the things I've changed beyond the changes in #1349
